modules/osd-cluster-wide-proxy.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * networking/proxies/configuring-cluster-wide-proxy.adoc
+
+[id="osd-cluster-wide-proxy-config_{context}"]
+= Configuring a cluster-wide proxy using {product-title}
+
+You can add a proxy to your cluster when you are creating a new one. Prior to installation, however, you should verify that the proxy is accessible from the intended cluster virtual private cloud (VPC) and its private subnets.
+
+.Procedure
+To create a cluster with a proxy, run the following command:
+[source, terminal]
+----
+ocm create cluster \
+ <other arguments here> \
+ --additional-trust-bundle-file <path-to-CA-bundle-file> \
+ --http-proxy http://<username>:<pswd>@<ip>:<port> \
+ --https-proxy https://<username>:<pswd>@<ip>:<port>
```

Comment:
`--https-proxy` field can also use **http**://<username>:<pswd>@<ip>:<port>.Do we need to explain?

Reply:
I think <other arguments here> is simple.User will confuse about how to configure

Reply:
Agree on the `https-proxy` point. Not sure what is the best way to represent that. 
Maybe
```suggestion
 --https-proxy http(s)://<username>:<pswd>@<ip>:<port>
```

Regarding the <other arguments here>, I also agree that it is simplistic, but I am not sure what is the best way to represent that other arguments will be required when creating a cluster (and we can't be able to predict what each customer's individual argument set may look like). Maybe we just point them at the install docs? (ie https://docs.openshift.com/rosa/rosa_getting_started/rosa-installing-rosa.html)

Reply:
Fixed the address with the `http(s)` change, but for the link, would this be the OSD version? https://docs.openshift.com/dedicated/osd_cluster_create/creating-your-cluster.html

I can add a note in the considerations section with a link to the respective documentation.

Reply:
If only link to the creating cluster url, it is still confused. I think we should point that user need prepare a CCS cluster with VPC  which the proxy can be accessible.